sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-b2c-resource-server/README.md

```diff
@@ -0,0 +1,168 @@
+  ```
+
+  While running sample, if error occurs with logs above:
+
+  - make sure that while creating user workflow by following this [guide](https://docs.microsoft.com/azure/active-directory-b2c/tutorial-create-user-flows), for **User attributes and claims** , attributes and claims for **Display Name** should be chosen.
+---
+- `Missing 'tenant-id' in application.yml`
+    
+    ```
+    Method springSecurityFilterChain in org.springframework.security.config.annotation.web.configuration.WebSecurityConfiguration required a bean of type 'org.springframework.security.oauth2.jwt.JwtDecoder' that could not be found.
+        The following candidates were found but could not be injected:
+        - Bean method 'jwtDecoder' in 'AADResourceServerConfiguration' not loaded because @ConditionalOnResource did not find resource 'classpath:aad.enable.config'
+        - Bean method 'jwtDecoder' in 'AADB2CResourceServerAutoConfiguration' not loaded because @ConditionalOnProperty (azure.activedirectory.b2c.tenant-id) did not find property 'tenant-id'
+        - Bean method 'jwtDecoderByIssuerUri' in 'OAuth2ResourceServerJwtConfiguration.JwtDecoderConfiguration' not loaded because OpenID Connect Issuer URI Condition did not find issuer-uri property
+        - Bean method 'jwtDecoderByJwkKeySetUri' in 'OAuth2ResourceServerJwtConfiguration.JwtDecoderConfiguration' not loaded because @ConditionalOnProperty (spring.security.oauth2.resourceserver.jwt.jwk-set-uri) did not find property 'spring.security.oauth2.resourceserver.jwt.jwk-set-uri'
+        - Bean method 'jwtDecoderByPublicKeyValue' in 'OAuth2ResourceServerJwtConfiguration.JwtDecoderConfiguration' not loaded because Public Key Value Condition did not find public-key-location property
+    ```
+
+    While running sample, if error occurs with logs above:
+    - `azure-activedirectory-b2c:tenant-id` should be added to the `application.yml`.
+---
+- `WWW-Authenticate: Bearer error="invalid_token", error_description="An error occurred while attempting to decode the Jwt: Couldn't retrieve remote JWK set: Read timed out",`
+  
+    While running sample, if error occurs with logs above:
+    - `azure-activedirectory-b2c:jwt-read-timeout` to set longer read time in `application.yml`.
+    
+### FAQ
+#### Sign in with loops to B2C endpoint ?
+This issue almost due to polluted cookies of `localhost`. Clean up cookies of `localhost` and try it again.
+
+#### More identity providers from AAD B2C login ?
+Follow the guide of [Set up Google account with AAD B2C](https://docs.microsoft.com/azure/active-directory-b2c/active-directory-b2c-setup-goog-app).
+And also available for Amazon, Azure AD, FaceBook, Github, Linkedin and Twitter.
+
+#### How do I delete or modify Application Permissions in Portal?
+You Can set `isEnabled` to `false` in the manifest's JSON configuration.Then delete or modify it.
+
```

Comment:
What's this?

Reply:
The method for modifying `roles` requires `isEnabled=false`.

Reply:
Could you please attach a screenshot here in the comment? I'm not familiar with this `isEnabled` still.